src/app/modules/react-migration/react-migration.controller.ts

```diff
@@ -216,9 +217,10 @@ export const servePublicForm: ControllerHandler<
   })
 
   if (showReact) {
-    return serveFormReact(
+    const isPublicForm = Boolean(
       !formResult.isErr() && formResult.value.status === FormStatus.Public,
-    )(req, res, next)
+    )
+    return serveFormReact(/* isPublic= */ isPublicForm)(req, res, next)
```

Comment:
dangling comment?

Reply:
removed in [00b75a7](https://github.com/opengovsg/FormSG/pull/5799/commits/00b75a7cc63e73d54ee8d1b501676332a0572be1)